jre_emul/jre_sources.mk

```diff
@@ -1247,7 +1247,22 @@ JAVA_PUBLIC_SOURCES_SECURITY = \
   javax/security/cert/CertificateExpiredException.java \
   javax/security/cert/CertificateNotYetValidException.java \
   javax/security/cert/CertificateParsingException.java \
-  javax/security/cert/X509Certificate.java
+  javax/security/cert/X509Certificate.java \
+  javax/sql/CommonDataSource.java \
+  javax/sql/ConnectionEvent.java \
+  javax/sql/ConnectionEventListener.java \
+  javax/sql/ConnectionPoolDataSource.java \
+  javax/sql/DataSource.java \
+  javax/sql/PooledConnection.java \
+  javax/sql/RowSet.java \
+  javax/sql/RowSetEvent.java \
+  javax/sql/RowSetInternal.java \
+  javax/sql/RowSetListener.java \
+  javax/sql/RowSetMetaData.java \
+  javax/sql/RowSetReader.java \
+  javax/sql/RowSetWriter.java \
+  javax/sql/StatementEvent.java \
+  javax/sql/StatementEventListener.java
```

Comment:
These need to go into the JAVA_PUBLIC_SOURCES_SQL list, not security sources.

Reply:
Done. Thanks for catching that